CircomIntro/step-3/README.md

```diff
@@ -0,0 +1,33 @@
+Now that you have the Circuit Compiler plugin installed, let's write a simple Circom circuit.
+
+## Creating a New Circuit File
+
+1. In the **File Explorer** on the left sidebar, click on the **Create New File** icon.
+2. Name your file `multiplier.circom` and press **Enter**.
+
+<img src="https://i.ibb.co/rMCHK4B/create-new-file.png" alt="create-new-file" width=300 height=200>
+
+## Writing the Circuit
+
+Open `multiplier.circom` and add the following code:
+
+```circom
+pragma circom 2.0.0;
+
+template Multiplier() {
+    signal input a;
+    signal input b;
+    signal output c;
+
+    c <== a * b;
+}
+
+component main = Multiplier();
+```
+
+## Explanation:
+- template `Multiplier()`: Defines a new circuit template called Multiplier.
+- `signal input a;` and `signal input b;`: Declare input signals a and b.
```

Comment:
there should maybe be some explanation about what is a signal and how is this different from a standard variable.